README.md

```diff
@@ -20,6 +16,10 @@ To populate the search index: `cfgov/manage.py update_index comparisontool`
 
 (this assumes you have a recent database dump)
 
+## Testing
+
+To run the unit tests, from the root directory of this repo: `tox`.
```

Comment:
We aren't consistent about documenting this, but for a newcomer to this repo it might be useful to explicitly say that the tests depend on having tox installed, with [a link to the tox documentation](https://tox.readthedocs.io/en/latest/). Otherwise developers might expect that a clone of this repository would be sufficient to run that command.

Reply:
Good catch. Added language for that in c4ebaad1f03cf43e06cfb93b9ffdf7ea2b4be9de.

Reply:
I'm definitely open to feedback on my wording, btw. (and everything else 😄)